.travis.yml

```diff
@@ -69,6 +66,8 @@ before_install:
 - export CHROME_BIN=/usr/bin/google-chrome-stable
 - export DISPLAY=:99.0
 - bash -e /etc/init.d/xvfb start
+- wget --no-verbose -O /tmp/$(basename $CHROME_SOURCE_URL) $CHROME_SOURCE_URL
```

Comment:
Just to check, what is CHROME_SOURCE_URL? Just wondering if it'll break in the future when Chrome updates again, and if it's possible to use something stable.

Reply:
I have mentioned it in the PR description. I found a repository which stores deb files of various chrome versions: https://github.com/webnicer/chrome-downloads

Reply:
Ah, sorry. Where are these "repository settings"? Should we document the procedure for updating to a newer version of Chrome here, in comments?

(We will have to do that eventually. But better that happen under our control, rather than under our feet and breaking us every time.)

Reply:
In case there is a problem with the link, we can directly change it by modifying this environment variable in Travis's repo settings.

Reply:
I think, in this file, you should provide instructions on how exactly to do that (in comments). It's the natural place that people will look.

Maybe in a separate PR after this one, though, since we want to get this one in as quickly as possible to unbreak develop.

Reply:
Here are the Travis repository settings: https://travis-ci.org/oppia/oppia/settings
The environment variables are under the same-named header.

Reply:
OK, thanks! I've merged this PR. Can you put these instructions in a comment here, as mentioned previously?

Reply:
OK, I will do this. Thanks!